src/mavlink-router/ulog.cpp

```diff
@@ -73,25 +89,9 @@ bool ULog::start()
 
 void ULog::stop()
 {
-    mavlink_message_t msg;
-    mavlink_command_long_t cmd;
-
-    if (_file == -1) {
```

Comment:
This check should probably still be in here.

Reply:
@bkueng The idea was to get away from assuming that log streaming can only be active if the logging file has a valid file descriptor. However, I reviewed the code and i don't think this case can happen, so I'm fine with putting that condition back in.

Reply:
When I went though it I think I saw a case where it could happen, though I don't remember exactly.
If called and no log file was initialized yet, [chmod](https://github.com/Auterion/mavlink-router/blob/a7f73cb377e498b526ffffd5be99ba8f90587784/src/mavlink-router/logendpoint.cpp#L156) would be called on an invalid file/something unexpected.